### DISCUSSION TITLE
Where is the documentation for the yellow-user.ini?

### DISCUSSION THREAD

**Discussion by sorenpeter** [datenstrom/community#496] [Ask a question] [answered]
Hej,

Will you update the documentation for user.ini ?

I was unclear to me how to restict access for user using the home.
First i tried to use the paths form the server such as, which did not work:
Home: /content/4-cph
Home: /4-cph

But i finally found that it is the exposed path in the url, that seem to do the trick:
Home: /cph

I would suggest that this info to be added to:
https://datenstrom.se/yellow/help/adjusting-system

And maybe more places:
https://github.com/datenstrom/yellow-extensions/tree/master/features/edit
https://github.com/datenstrom/yellow-extensions/blob/a556c8dcebeb1d2261b4c2756fbeba77a7a4433c/features/edit/README.md
https://github.com/datenstrom/yellow-extensions/blob/e183b2f549922cd1b3742b3631ab929ff7ebbefc/features/help/en/security-configuration.md

/Søren Peter

**Comment by markseuffert** [comment, marked as answer]
Hej, you can find it here:

https://github.com/annaesvensson/yellow-edit

Thanks for the feedback. We are in the process of updating the documentation, that’s why it’s hard to find at the moment.
